pkg/dispatcher/dispatcher_test.go

```diff
@@ -17,13 +17,15 @@ limitations under the License.
 package dispatcher
 
 import (
+	"fmt"
 	"runtime"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/apache/incubator-yunikorn-k8shim/pkg/common/utils"
```

Comment:
all yunikorn imports should be grouped together. I filed new issues to fix the linter setting but this should be with the events import (k8shim  #75)

Reply:
Done.